### DISCUSSION TITLE
cannot access <ip>:64297 (tpot dashboard)

### DISCUSSION THREAD

**Discussion by hareeshrk7** [telekom-security/tpotce#1622] [Q&A]
i start my tpot as normal i do. but tdy i cannot acces the tpot dashboard. samore my "dps.sh" cannot be found instead there have a file called "dps.ps1" can anyone help me with this

**Comment by RytterMohn** [comment]
hello I got same problem and I have solved it.
I use debian system and use ./install.sh to install tpot.
you can use 
```
docker ps -a
```
to see what happend to your docker. 
And you will find your docker containers are restart endless.
that's because your port 631 has been allocated to other service.
I written my solve method in here:[Docker containers seem to restart endless #1583](https://github.com/telekom-security/tpotce/discussions/1583)
#1583 
I am sure you can solve this problem by this method.
